tests/ats/test_app.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from typing import List, Tuple
+
+import pytest
+import pykube
+from pytest_helm_charts.clusters import Cluster
+from pytest_helm_charts.k8s.deployment import wait_for_deployments_to_run
+from pytest_helm_charts.k8s.stateful_set import wait_for_stateful_sets_to_run
+
+
+logger = logging.getLogger(__name__)
+
+namespace_name = "loki"
+write_sts_name = "loki-write"
```

Comment:
```suggestion
write_statefulset_name = "loki-write"
```

You know the drill 😅